ui/src/app/kvm/components/CoreResources/CoreResources.tsx

```diff
@@ -1,27 +1,25 @@
 import classNames from "classnames";
 
-import PodMeter from "app/kvm/components/PodMeter";
+import KVMResourceMeter from "app/kvm/components/KVMResourceMeter";
 import { getRanges } from "app/utils";
 
 export type Props = {
-  cores: {
-    allocated: number;
-    free: number;
-  };
+  allocated: number | number[];
   dynamicLayout?: boolean;
-  pinned?: number[];
-  available?: number[];
+  free: number | number[];
+  other?: number;
 };
```

Comment:
Since providing pinned or unpinned cores is mutually exclusive, I've made the relevant props accept two different types instead of having two sets of props. The component then determines how to handle whether the cores have been given as an array (pinned) or not (unpinned)